src/surge-xt/osc/OSCListener.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+** Surge Synthesizer is Free and Open Source Software
+**
+** Surge is made available under the Gnu General Public License, v3.0
+** https://www.gnu.org/licenses/gpl-3.0.en.html
+**
+** Copyright 2004-2021 by various individuals as described by the Git transaction log
+**
+** All source at: https://github.com/surge-synthesizer/surge.git
+**
+** Surge was a commercial product from 2004-2018, with Copyright and ownership
+** in that period held by Claes Johanson at Vember Audio. Claes made Surge
+** open source in September 2018.
+*/
+
+#include "OSCListener.h"
+#include "Parameter.h"
+#include "SurgeSynthProcessor.h"
+#include <sstream>
+#include <vector>
+#include <string>
+
+
+using namespace Surge::OSC;
```

Comment:
All this stuff is supposed to be inside of Surge::OSC. So instead, what you need to do is this:

namespace Surge {
namespace OSC {

// ... rest of the file

}  // namespace OSC
}  // namespace Surge